lib/commands/all.js

```diff
@@ -1,32 +1,26 @@
-import * as _ from 'lodash';
+import _ from 'lodash';
 
-import DatabaseClient from '../database/database-client';
 import logger from '../utilities/logger';
-import { readMigrationFiles } from '../migrations/migration-files';
 import { pendingMigrations } from '../migrations/migration-filters';
+import command from './command';
```

Comment:
Could we name this function something a little more descriptive?

Reply:
Seconded, I was googling for a `command` lib at first. Ideas, no of which I am attached to: `executeMigration`, `execute`, `migrate`, `executeCommand`